src/main/resources/beam-template.conf

```diff
@@ -93,6 +93,15 @@ beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.walk_intercept = "do
 beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.bike_intercept = "double | 0.0"
 beam.agentsim.agents.modalBehaviors.mulitnomialLogit.utility_scale_factor = "double | 1.0"
 beam.agentsim.agents.modalBehaviors.lccm.filePath = ${beam.inputDirectory}"/lccm-long.csv"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.generate_secondary_activities = "false"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.intercept_file_path = ${beam.inputDirectory}"/activity-intercepts.csv"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.activity_file_path = ${beam.inputDirectory}"/activity-params.csv"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.additional_trip_utility = "double | 0.0"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.max_destination_distance_meters = "double | 32000"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.max_destination_choice_set_size = "int | 20"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.destination_nest_scale_factor = "double | 1.0"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.mode_nest_scale_factor = "double | 1.0"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.trip_nest_scale_factor = "double | 1.0"
```

Comment:
These three values should all be between zero and one and determine the amount of noise in each level of the nested choice process. Increasing destination_nest_scale_factor means that people are more likely to choose a less optimal destination, mode_nest_scale_factor means people are more likely to value destinations accessible by multiple modes, and trip_nest_scale_factor means that people are more likely to take secondary trips even if the costs are greater than the benefits